### PR TITLE
Update docker, influxdb and postresql protoboards

### LINKED ISSUE #4792: update influxdb protoboard
[InfluxDB.json.zip](https://github.com/influxdata/chronograf/files/2567817/InfluxDB.json.zip)

### PR DESCRIPTION
**Pull request by ebb-tide**
Closes #4792 #4783 #4792

### CODE REVIEW

File: protoboards/influxdb.json [hunk 1]
```diff
@@ -8,302 +8,2165 @@
...
+              "measurement": "",
+              "retentionPolicy": "",
+              "fields": [],
+              "tags": {},
+              "groupBy": {
+                "time": "",
+                "tags": []
+              },
+              "areTagsAccepted": false,
+              "rawText": "SELECT non_negative_derivative(max(\"bytes_recv\"), 1s) AS \"recieved\", non_negative_derivative(max(\"bytes_sent\"), 1s) AS \"sent\" FROM \":db:\".\":rp:\".\"net\" WHERE time > :dashboardTime: AND \"host\" = :host: GROUP BY time(:interval:) FILL(null)",
```
Comment (russorat) [inline protoboards/influxdb.json]: received

File: protoboards/influxdb.json [hunk 2]
```diff
@@ -8,302 +8,2165 @@
...
+      },
+      {
+        "x": 5,
+        "y": 1,
+        "w": 3,
+        "h": 3,
+        "name": "Network",
+        "queries": [
+          {
+            "query": "SELECT non_negative_derivative(max(\"bytes_recv\"), 1s) AS \"recieved\", non_negative_derivative(max(\"bytes_sent\"), 1s) AS \"sent\" FROM \":db:\".\":rp:\".\"net\" WHERE time > :dashboardTime: AND \"host\" = :host: GROUP BY time(:interval:) FILL(null)",
```
Comment (russorat) [inline protoboards/influxdb.json]: received

File: protoboards/influxdb.json (hunk 1 above)
Comment (ebb-tide) [reply protoboards/influxdb.json]: nice catch!

**Comment by ebb-tide** [comment]
Cherry picked in to 1.7.x

**Pull request merged**